app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/Claim.scala

```diff
@@ -31,8 +32,8 @@ final case class Claim(
 )
 
 object Claim {
-  implicit class ListClaimOps(private val claims: List[Claim]) {
-    def total: BigDecimal = claims.map(c => c.claimAmount).sum
+  implicit class ListClaimOps(private val claims: NonEmptyList[Claim]) {
+    def total: BigDecimal = claims.map(c => c.claimAmount).toList.sum
```

Comment:
do we use this in the BE? it was added in the FE so calculate the totals in the summary page

Reply:
Yes we do: ClaimTransformerService line 82 and 156